### PR TITLE
Fix test_alg_info.py on Windows platform

### PR DESCRIPTION
**Pull request by qnfm**
Fix test_alg_info.py on Windows which cause the ninja run_tests failure.

Since `gerp` is not available on Windows, making use of powershell is a good alternative way to replace grep.

### CODE REVIEW

File: tests/test_alg_info.py [hunk 1]
```diff
@@ -66,5 +74,6 @@ def test_alg_info_sig(sig_name):
 
 if __name__ == "__main__":
     import sys
+    print("platform:",platform.system)
```
Comment (baentsch) [inline tests/test_alg_info.py:77]: Is this necessary?

**Review by baentsch — APPROVED** [review]
LGTM. Thanks for bringing in Windows knowhow!

File: tests/test_alg_info.py (hunk 1 above)
Comment (qnfm) [reply tests/test_alg_info.py:77]: Aha, forgot to remove. My fault.

**Review by dstebila — APPROVED** [review]

**Review by SWilson4 — APPROVED** [review]
Thank you very much for the contribution!

**Pull request merged**